### PR TITLE
[DX-1560] Update latest LTS release

### PR DESCRIPTION
**Pull request by dcs3spp**
### For internal users - Please add a Jira DX PR ticket to the subject!
[DX-1560](https://tyktech.atlassian.net/browse/DX-1560)

---

### Preview Link

[preview - Release notes landing page](https://deploy-preview-5125--tyk-docs.netlify.app/docs/nightly/developer-support/tyk-release-summary/overview)
[preview - Dashboard release notes overview](https://deploy-preview-5125--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/overview)
[preview - GW release notes overview](https://deploy-preview-5125--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-gateway/release-notes/overview)

### Description

- Update release notes to latest 5.3.x LTS version
- Update release notes overview pages for GW & Dashboard to mention 5.3.x LTS

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labeled** the PR

[DX-1560]: https://tyktech.atlassian.net/browse/DX-1560?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

___

### **PR Type**
Documentation, Enhancement

___

### **Description**
- Updated LTS version formatting in the release summary to include "LTS".
- Added LTS designation to Tyk Dashboard 5.3.x release notes.
- Added LTS designation to Tyk Gateway 5.3.x release notes.
- Updated LTS version to 5.3.2 for Tyk Dashboard.
- Updated LTS version to 5.3.2 for Tyk Gateway.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>release-summary.html</strong><dd><code>Update LTS version formatting in release summary</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/themes/tykio/layouts/partials/developer-support/latest-releases/release-summary.html

- Updated LTS version formatting to include "LTS" in the title.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5125/files#diff-b8667e2938c92c61d06de8b8c86b65ed48f30886c84500e65348975f87c8822a">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>dashboard.json</strong><dd><code>Update Tyk Dashboard LTS version to 5.3.2</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/data/releases/dashboard.json

- Updated LTS version to 5.3.2 for Tyk Dashboard.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5125/files#diff-3bd5dbfe246a860bb9ba2a172f0b0cc23b24de6971b5257e59387e77d2f4fa4b">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>gateway.json</strong><dd><code>Update Tyk Gateway LTS version to 5.3.2</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/data/releases/gateway.json

- Updated LTS version to 5.3.2 for Tyk Gateway.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5125/files#diff-d5c41dd0d75af9a26732cba20aaad8331f7c995224e0a5d8b9f311bb218197ce">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>overview.md</strong><dd><code>Add LTS designation to Tyk Dashboard 5.3.x release notes</code>&nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-dashboard/release-notes/overview.md

- Updated release note link for version 5.3.x to include "LTS".

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5125/files#diff-85e62f3ad4c1b71593a3c7aedf6d4eb27d69b5034ea907496a9dd9e0a65360dd">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>overview.md</strong><dd><code>Add LTS designation to Tyk Gateway 5.3.x release notes</code>&nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-gateway/release-notes/overview.md

- Updated release note link for version 5.3.x to include "LTS".

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5125/files#diff-70d23a1afc5233df58dd45c9bee1f32ef180620854eafa1cfa2984effe9ffe0d">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

**Review by letzya — CHANGES_REQUESTED** [review]
I think this page should also reflect the fact that it’s lts https://deploy-preview-5125--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.3/

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/overview.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This page provides access to release notes for Tyk Gateway. Links to archived re
 ## Release 5
 
 - [v5.4]({{< ref "product-stack/tyk-gateway/release-notes/version-5.4.md" >}})
-- [v5.3]({{< ref "product-stack/tyk-gateway/release-notes/version-5.3.md" >}})
+- [v5.3.x]({{< ref "product-stack/tyk-gateway/release-notes/version-5.3.md" >}}) - LTS
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/overview.md]:
```suggestion
- [v5.3]({{< ref "product-stack/tyk-gateway/release-notes/version-5.3.md" >}}) - LTS
```

Comment (letzya) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/overview.md]: Why do we need `.X`?

Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/overview.md]: updated

**Comment by dcs3spp** [comment]
> I think this page should also reflect the fact that it’s lts https://deploy-preview-5125--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.3/

updated

**Review by letzya — APPROVED** [review]

**Pull request merged**